.travis.yml

```diff
@@ -75,10 +75,7 @@ install: |-
     osx)
       MACOSX_DEPLOYMENT_TARGET=$DIST
       brew update # see travis-ci/travis-ci#8552
-      for p in gettext libtool automake zlib;
-        do brew list $p >/dev/null || brew install --verbose $p; done
-      brew link --force gettext libtool zlib
-      brew install --verbose wxmac --with-static --with-stl
+      brew bundle
```

Comment:
Thanks.
I'll add `--file` to move spec to subdir and `--verbose` to prevent travis timeout.

Reply:
sure. which subdir do you want?

Reply:
/util/

Reply:
done